package.json

```diff
@@ -40,16 +40,16 @@
     "webpack-dev-server": "^4.7.3"
   },
   "dependencies": {
-    "@types/ws": "^8.5.0",
```

Comment:
It looks like this type was resolved to `version "8.5.13" in the orum repo previously?